contrib/hydra/docker-compose.yml

```diff
@@ -0,0 +1,133 @@
+# This docker-compose file sets up ORY Kratos, ORY Hydra, and this app in a network and configures
+# in such a way that ORY Kratos is the Login Provider for ORY Hydra.
+
+version: '3.7'
+
+services:
+  postgresd:
+    image: postgres:9.6
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_USER=pguser
+      - POSTGRES_PASSWORD=secret
+      - POSTGRES_MULTIPLE_DATABASES=kratos,hydra
+    volumes:
+      - ./pg-init:/docker-entrypoint-initdb.d
+    networks:
+      - intranet
+
+  hydra-migrate:
+    image: oryd/hydra:v1.8.5-alpine
+    depends_on:
+      - postgresd
+    environment:
+      - DSN=postgres://pguser:secret@postgresd:5432/hydra?sslmode=disable
```

Comment:
We could also use SQLite here and remove the need for pg-init et al?

Reply:
Yeah, no idea why this was on postgres, figured the original committer had a reason